packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -224,7 +224,7 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
           // root props has been deprecated and should get removed.
           // it needs to be marked as "any" since root props expects a div element, but really Tag can
           // be any native element so typescript rightly flags this as a problem.
-          ...rootProps as any
+          ...(rootProps as any)
```

Comment:
This change seems to be made automatically. I'm not sure what's causing this.

Reply:
Its caused automatically by the Prettier configuration checked into the repo.